samples/net/eth_native_posix/README.rst

```diff
@@ -28,15 +28,24 @@ below.
    :compact:
 
 Normally one needs extra privileges to create and configure the TAP device in
-the host system. Use sudo to execute the Zephyr process with admin privileges,
-like this:
+the host system. If the user has set the
+:option:`CONFIG_ETH_NATIVE_POSIX_STARTUP_AUTOMATIC` option (this is disabled
+by default), then the user needs to use ``sudo`` to execute the Zephyr process
+with admin privileges, like this:
 
 .. code-block:: console
 
     sudo --preserve-env=ZEPHYR_BASE make run
 
 If the ``sudo --preserve-env=ZEPHYR_BASE`` gives an error,
-just use ``sudo --preserve-env`` instead:
+just use ``sudo --preserve-env`` instead.
+
+If that option is not enabled (this is the default), then the user should
+execute the ``net-setup.sh`` script from Zephyr `net-tools`_ repository.
+The script should be run before executing the Zephyr process. The script
+will create the zeth interface and setup IP addresses and routes.
```

Comment:
```suggestion
will create the zeth interface and set up IP addresses and routes.
```

Reply:
FYI, ``setup`` is a noun or adjective (e.g., the setup script), while ``set up`` is the verb form.

Reply:
Continue with the command example that was in the Kconfig file:

    You can run the ``net-setup.sh`` script like this::

       cd net-tools
       ./net-setup.sh

    or::

       ./net-setup.sh --config ./zeth-vlan.conf